packages/commonwealth/client/scripts/views/components/Comments/CreateComment.tsx

```diff
@@ -171,7 +171,12 @@ export const CreateComment = ({
             fontWeight="medium"
             className={clsx('user-link-text', { disabled: !canComment })}
           >
-            <User user={author} hideAvatar linkify />
+            <User
+              userAddress={author.address}
```

Comment:
This change is the cause of the error in the e2e test -- author is undefined, probably because it loads the component without the proper value of `app.user.activeAccount`. Please fix if possible!

Also if you search for all uses of
```
  const author = app.user.activeAccount;
```
You will see several other pages where this may be a problem.

Reply:
@jnaviask I am not getting undefined author. Tried with these steps

---
#### Secnario # 1

1. Login and go to a joined community (I tried with dydx)
2. Open a thread
3. Add a comment

-- 

#### Secnario # 1

1. Go to a joined community, and open a thread (I tried with dydx)
2. Logout and then login
3. Add a comment

--

> This change is the cause of the error in the e2e test 

Hmm, I am not sure about this, most of them seem to be sequelize errors?